encoding/dot/topgraph.go

```diff
@@ -9,8 +11,8 @@ import (
 	internalEncoding "gorgonia.org/gorgonia/internal/encoding"
 )
 
-var (
-	subGraphs = map[internalEncoding.Group]subgrapher{
```

Comment:
All my life, i’ve been told that global variables are evil.... I am trying to convince myself that they are not. I guess my teacher were right :)